docs/changes.md

```diff
@@ -1,6 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+
+- `12.9` Added JDBC configuration to connect to an external SSL and not SSL enabled database for Manage application (DB2, Oracle and SQL Server) ([#600]( https://github.com/ibm-mas/ansible-devops/pull/600))
```

Comment:
Entry in changelog should match the PR title `Add JDBC configuration to connect to external databases`, also needs adding to the duplicate changelog that shows up in ansible galaxy (see `ibm/mas_devops/README.md`)

Reply:
Updated changelogs.